### PR TITLE
feat: Enable symbol importing in `@PhpCsFixer` ruleset

### PR DESCRIPTION
**Pull request by Wirone**
In general I believe it should be enabled for our codebase, the only place where it produces large overhead is `tests/Test/AbstractFixerTestCase.php` but this list of tests should shrink in time so amount of imports will be reduced too.

### CODE REVIEW

**Comment by Wirone** [comment]
@nicolas-grekas are you interested in importing FQCNs in `@Symfony` ruleset? Then I can move the configuration and `@PhpCsFixer` will inherit it 🙂.

**Comment by nicolas-grekas** [comment]
Naively I'd say yes but on second thought we have symbols that are not imported on purpose and we should have very few occurrences that need an actual fix.
Let's not for now, then run the rule on the codebase so we can consider what we want.

**Comment by Wirone** [comment]
Rule is already available, you just need to run Fixer with `--rules='{"fully_qualified_strict_types":{"import_symbols":true}}"` to verify the range of changes 🙂.

File: src/RuleSet/Sets/PhpCsFixerSet.php [hunk 1]
```diff
@@ -55,6 +55,9 @@ public function getRules(): array
             'escape_implicit_backslashes' => true,
             'explicit_indirect_variable' => true,
             'explicit_string_variable' => true,
+            'fully_qualified_strict_types' => [
+                'import_symbols' => true,
+            ],
```
Comment (keradus) [inline src/RuleSet/Sets/PhpCsFixerSet.php:60]: FYI, changing rule configuration is not DX, it's feat.

changing `.php-cs-fixer.dist.php` config is DX/chore

File: src/Console/Output/ErrorOutput.php [hunk 2]
```diff
@@ -86,7 +87,7 @@ public function listErrors(string $process, array $errors): void
                 $this->output->writeln('');
                 $stackTrace = $e->getTrace();
                 foreach ($stackTrace as $trace) {
-                    if (isset($trace['class']) && \Symfony\Component\Console\Command\Command::class === $trace['class'] && 'run' === $trace['function']) {
+                    if (isset($trace['class']) && Command::class === $trace['class'] && 'run' === $trace['function']) {
```
Comment (keradus) [inline src/Console/Output/ErrorOutput.php:90]: can you describe why we needed to add isset?
it was some warning in phpstan, or we changed the logic somewhere?

File: tests/Test/AbstractFixerTestCase.php [hunk 3]
```diff
@@ -30,6 +30,72 @@
 use PhpCsFixer\PhpunitConstraintIsIdenticalString\Constraint\IsIdenticalString;
 use PhpCsFixer\Preg;
 use PhpCsFixer\StdinFileInfo;
+use PhpCsFixer\Tests\Fixer\ClassNotation\ClassAttributesSeparationFixerTest;
```
Comment (keradus) [inline tests/Test/AbstractFixerTestCase.php:33]: yey 😅

**Review by keradus — APPROVED** [review]

File: src/Console/Output/ErrorOutput.php (hunk 2 above)
Comment (Wirone) [reply src/Console/Output/ErrorOutput.php:90]: `isset()` was already there, the only changes in this PR are: ruleset reconfiguration (manual) and applying CS fixes (automated). Here FQCN was shortened, nothing more.

Comment (keradus) [reply src/Console/Output/ErrorOutput.php:90]: 🤦🏻 with split-view, I misseen the check as it was visually broken to line above 🤦🏻 

Screenshot 2023-12-30 at 01 18 08

**Pull request merged**